api/metrics/src/main/java/io/opentelemetry/api/metrics/internal/StringUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.metrics.internal;
+
+import javax.annotation.concurrent.Immutable;
+
+/** Internal utility methods for working with attribute keys, attribute values, and metric names. */
+@Immutable
+public final class StringUtils {
```

Comment:
Can we call this `MetricStringUtils`? I know we would also want to rename some of our current classes, but it makes the code more readable I think and the IDE more friendly for development.

Reply:
sure, we will probably move back anyway when getting close to release.

Reply:
Done.